JavaScript/Endpoint Examples/Multipart Payload/pdf-with-redacted-text-preview.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * This request demonstrates how to apply text redaction previews to a PDF.
+ * This is step 1 of 2 toward applying redactions to text.
+ */
+var axios = require('axios');
+var FormData = require('form-data');
+var fs = require('fs');
+
+// Create a new form data instance and append the PDF file and parameters to it
+var data = new FormData();
+data.append('file', fs.createReadStream('/path/to/file'));
+var redaction_option_array = [];
+var redaction_options1 = {
+    "type": "preset",
+    "value": "uuid",
```

Comment:
```suggestion
    "value": "email",
```

I get this error for this set of options. Replacing with `email` (or some other valid option) worked.

```
{"error": "uuid is not a valid preset redaction option. See documentation for valid options."}
```